projects/packages/videopress/src/client/components/video-upload-area/style.module.scss

```diff
@@ -0,0 +1,64 @@
+.wrapper {
+	max-width: 552px;
+	width: 100%;
+	border-radius: var(--jp-border-radius);
+	border: 1px dashed var(--jp-gray-20);
+	background-color: var(--jp-gray-0);
+	padding: 48px 0 64px;
```

Comment:
For `padding` and `margin`, we use `var( --spacing-base )` and multiplies ( with `calc` ), that will make use keep the [8px grid](https://uxplanet.org/everything-you-should-know-about-8-point-grid-system-in-ux-design-b69cb945b18d)